src/repositories/Repositories.ts

```diff
@@ -17,7 +17,7 @@
 import { Sequelize, InitOptions } from 'sequelize';
 
 import StakeRequestRepository from './StakeRequestRepository';
-import { AuxiliaryChainRepository } from './AuxiliaryChainRepository';
+import  AuxiliaryChainRepository  from './AuxiliaryChainRepository';
 import { MessageRepository } from './MessageRepository';
```

Comment:
The import for all repos should be like in StakeRequestRepository (without {} braces). As there is only a single default export.

Reply:
Agree! Made the changes.
Only MessageRepository can't be default export as MessageModel of MessageRepository is a dependency for StakeRequestRepository.
```
StakeRequestModel.belongsTo(MessageModel, { foreignKey: 'messageHash' });
```

Reply:
MessageModel should not be dependency for StakeRequestRepository. StakeRequestRepository should only depend on Message (from model directory) and MessageRepository.

Reply:
It will not work because of type mismatch. It needs sequelize `MessageModel extends Model` type.
```
error TS2322: Type '(typeof Message)[]' is not assignable to type 'Includeable[]'.
  Type 'typeof Message' is not assignable to type 'Includeable'.
    Type 'typeof Message' is missing the following properties from type 'typeof Model': tableName, primaryKeyAttribute, associations, options, and 36 more.
``